packages/grafana-ui/src/components/Select/Select.story.tsx

```diff
@@ -240,12 +247,13 @@ export const BasicSelectAsync = () => {
       onChange={(v) => {
         setValue(v);
       }}
-      {...getDynamicProps()}
+      prefix={getPrefix(args.icon)}
+      {...args}
     />
   );
 };
 
-export const AutoMenuPlacement = () => {
+export const AutoMenuPlacement: Story = (args) => {
```

Comment:
Not sure about this, but shouldn't this story have the menuplacement set to auto?